homeassistant/components/smartthings/climate.py

```diff
@@ -20,6 +20,7 @@
     HVAC_MODE_COOL,
     HVAC_MODE_DRY,
     HVAC_MODE_FAN_ONLY,
+    HVAC_MODE_WIND,
```

Comment:
This constant does not exist. How did you test this?

Reply:
I thought those where inherited from the smart things python library. I've just tested the payload to the smart thing API.

Reply:
It says: `from homeassistant.components.climate.const import (`...